lib/arclight/custom_document.rb

```diff
@@ -81,5 +92,9 @@ def creators
     def access_subjects
       subjects_array(%w[subject function occupation genreform], parent: 'archdesc')
     end
+
+    def access_subjects_coll
+      subjects_array(%w[subject function occupation genreform], parent: "archdesc[@level='collection']")
```

Comment:
Per Slack discussion, I'm not sure this is necessary. `access_subjects` gets us what we need already.